src/components/Disputes/actions/DisputeAppeal.js

```diff
@@ -24,8 +24,22 @@ function DisputeAppeal({ onRequestAppeal, confirm }) {
       >
         {actionLabel}
       </Button>
-      <Info mode="description">
-        Anyone holding the required appeal deposit can trigger this action.
+      <Info>
+        <strong>Anyone</strong> can{' '}
+        <strong>
+          lock DAI as collateral to{' '}
+          {confirm ? 'confirm an appeal' : 'propose an appeal'}{' '}
+        </strong>
+        , if they don’t agree with the current outcome. When the final ruling is
```

Comment:
```suggestion
        if they believe the current outcome is incorrect. When the final ruling is
```

However, in the case of the person confirming the appeal, what they're actually doing is siding with the current outcome, no?

The appealer believes the ruling is incorrect; the confirmer invests against the appealer (and hence believes the current ruling is correct).

Reply:
I think that that would be the most common case yes.
However the account confirming an appeal can appeal for any ruling that is opposed to the ruling the appealer ruled for. So e.g let's say that the jury vote outcome was `In Favor`, someone doesn't agree with this outcome so he/she appeals `Against`. The one that confirms the appeal can rule for `In favor` (most common case) or `Refuse`. I don't see the reason of someone ruling for the later option  but it could happen.

Reply:
Have changed the wording for `if they believe the ruling appealed for is incorrect` in the case that the action is confirm appeal. Let me know what do you think

Reply:
Sounds good; I also don't know about `Refuse` although in the current situation with the first precedence campaign dispute, someone could actually appeal for "refuse" to help invalidate the ruling.